apps/hubble/src/rpc/httpServer.ts

```diff
@@ -578,9 +578,9 @@ export class HttpAPIServer {
 
     // @doc-tag: /relationshipsByRelatedObjectRef?ref_type=Cast/Object/Fid,object_ref_network=...&object_ref_fid=...&object_ref_hash=...&ref_direction=Source/Target&type=...
     this.app.get<{ Querystring: {
-      ref_type: number,
-      object_ref_network: number, object_ref_fid: number, object_ref_hash: string,
-      ref_direction: number,
+      ref_type: string,
+      object_ref_network: string, object_ref_fid: string, object_ref_hash: string,
+      ref_direction: string,
```

Comment:
these values don't come as anything but strings when a client actually calls the http endpoint. Need to parse everything out.